### PR TITLE
all registration and notification work

### PR DESCRIPTION
**Pull request by heshambadawy**
@team  Heare's All walker register and notification subscribtion

### CODE REVIEW

File: ios/App/App/capacitor.config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"appId": "com.aychihuahua.walkerboys",
-	"appName": "Aychihuahua",
+	"appName": "Aychihuahuaa",
```
Comment (Aychihuahuausa) [inline ios/App/App/capacitor.config.json:3]: remove last a

File: src/app/account/account.service.ts [hunk 2]
```diff
@@ -78,32 +78,32 @@ export class AccountService {
...
   acceptTerms() {
-    return this.http.post<CheckReponse[]>(this._url + '/acceptWalkerTerms', { clientId: this.currentClient.clientId })
+    return this.http.post<CheckReponse[]>(this._url + '/acceptWalkerTerms', { walkerId: this.currentClient[0].walkerId })
       .map(request => {
         return request;
       })
   }
   resendVarificationCode() {
-    return this.http.post<CheckReponse[]>(this._url + '/resendWalkerVarificationCode', { clientId: this.currentClient.clientId })
+    return this.http.post<CheckReponse[]>(this._url + '/resendWalkerVarificationCode', { clientId: this.currentClient.walkerId })
```
Comment (Aychihuahuausa) [inline src/app/account/account.service.ts:87]: change clientId to walkerId

File: src/app/account/account.service.ts [hunk 3]
```diff
@@ -78,32 +78,32 @@ export class AccountService {
...
       .map(request => {
         return request;
       })
   }
 
   signInWithExistingNo() {
-    return this.http.post<CheckReponse[]>(this._url + '/signInWalkerWithExistingNo', { clientId: this.currentClient.clientId })
+    return this.http.post<CheckReponse[]>(this._url + '/signInWalkerWithExistingNo', { clientId: this.currentClient.walkerId })
```
Comment (Aychihuahuausa) [inline src/app/account/account.service.ts:94]: also here

File: src/app/account/account.service.ts [hunk 4]
```diff
@@ -78,32 +78,32 @@ export class AccountService {
...
       .map(request => {
         return request;
       })
   }
   activateClient() {
-    return this.http.post<UserResponse>(this._url + '/activateWalker', { clientId: this.currentClient.clientId })
+    return this.http.post<UserResponse>(this._url + '/activateWalker', { clientId: this.currentClient.walkerId })
```
Comment (Aychihuahuausa) [inline src/app/account/account.service.ts:100]: also

File: src/app/main-client/notification-touch/notification-touch.component.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
...jozjUl1ukq2WJMma41/SVnfu3PyWPpxUN/SOrQhLDzXV3p90LLTmhuzFd6v1++4a1sBUYkU19nHRLUp8ZHI5otenEKQFBLcmlq23ssAF70/JgpCzqGrv2Ln4m5lPmTxXqPsL103dOPcb3QZY9zXm3tPau3gABV19heopptIFusAR5o0IddV/wBQTaZ1a2Vj4Z70T7O+3yahgrgKT9qrPtz1nS3kIQgt2ivPbDvaIYTFZhPp73TnFarw3S+3V1UUbuBWUKRhVevGab9GIcgMcUiOaP0RivEpWjmS2egae0oXGKX6xjOKWr1nYkTSg+0A3Q1elGS4Uy6eh5duYzS7WtihtR1VWHu0NZ6iJ96uB/HjztC+SC1rrtp5RSaZtq7t+Cwij+n3LVw45p5b6SIkV3rGuHFFkk0IrGkYZJpjpLxQ00Tpm6iH6NiuX9nvkjcYro3a6ysCan1XXrbKFFVbqWldWgUClh5zV1yiqZRJFqsm3cfc5xUt7pSXD7nAqri+U5NT6fr+zvAqkZXpoDXoZf7e6Xl2sI7zTDqlhdpyJiql1D2k3ONkn5UUOoM6yQadtRVAStiPU6rY5XJHmmugsq4kqTQ13bMkUw03UQogCkUt9DcV7Jz0eSIwDU2p9mhA2tXB6vjiom6s3rTuX4Fom/2tUHvZpfqOiG48I0L3qa51Mnmol6qVMityQGgXV9DGnIdCS4o/TdU1NyA/w1Fc6vu5ojSa2fSjysFDzTaxUX3omo9V1VH5NJriF6O0vSl2y1bbDowdTQd6C1OqR2Bniu+p9L92VoLR9DdwQDmjGwMN/wB1sjGCRVU9ptazqdggUx1nQf4Y7nM0o611a3s2iKpy8C0VfQWDcuAeteov0JPwBIzFeXdP1gS4G9au2p9sVKBQayZhTe6GNxrK4b2iXzWUNGJbdFW63WV4MuyC7B9ZxSN+a3WV6MP/AAivgYafisasrKmIH9A/+QV6dpvhFZWV1x6LR6CdLRzcVlZToYr+t+Kl2r4rdZSPoK7K5rqUa74RW6ylXaGfRZPZe0pQSoP0FWlrSx8I+wrKyqvomit9VUeKXpxWVlT8jeCTS80Ve7VlZTsBxqeBUKcVlZQMQfmqdeayspkYdaHimw4FarKzMSaj4BXXSOTWVlDyYRe3vwV4xrvjNZWUy7FYNb5qe5WVlFhQPWVlZQMf/9k=' alt=''>
```
Comment (Aychihuahuausa) [inline src/app/main-client/notification-touch/notification-touch.component.html:5]: add image to asset

File: src/app/model/client.ts [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import { Position } from "./position";
 
-export class Client {
-  clientId: number;
+export class Walker {
+  walkerId: number;
```
Comment (Aychihuahuausa) [inline src/app/model/client.ts:4]: keep client class we could need it on the future we have walker.ts plz use it

File: ios/App/App/capacitor.config.json (hunk 1 above)
Comment (heshambadawy) [reply ios/App/App/capacitor.config.json:3]: that is intended to be like  that  , to distinguish the two apps from each other

File: src/app/account/account.service.ts (hunk 3 above)
Comment (heshambadawy) [reply src/app/account/account.service.ts:94]: What here ???

File: src/app/main-client/notification-touch/notification-touch.component.html (hunk 5 above)
Comment (heshambadawy) [reply src/app/main-client/notification-touch/notification-touch.component.html:5]: No need to be asset i only have it as base 64 that just placeholder until we have a real design for notification

File: src/app/account/account.service.ts (hunk 2 above)
Comment (heshambadawy) [reply src/app/account/account.service.ts:87]: ahhh okay

File: src/app/account/account.service.ts (hunk 4 above)
Comment (heshambadawy) [reply src/app/account/account.service.ts:100]: ok

File: src/app/account/account.service.ts (hunk 3 above)
Comment (Aychihuahuausa) [reply src/app/account/account.service.ts:94]: change clientId >> walkerId

**Review by Aychihuahuausa — APPROVED** [review]

**Pull request merged**